lib/jxl/enc_frame.cc

```diff
@@ -327,6 +328,8 @@ Status MakeFrameHeader(size_t xsize, size_t ysize,
   if (jpeg_data) {
     // we are transcoding a JPEG, so we don't get to choose
     frame_header->encoding = FrameEncoding::kVarDCT;
+    frame_header->x_qm_scale = 2;
+    frame_header->b_qm_scale = 2;
```

Comment:
I am not sure why these lines got added here...

Reply:
I moved these so that most of the code that sets frame header fields is in this function and then when we have to change it later based on image data those can be guarded by update_global_state flag.